package.json

```diff
@@ -1,79 +1,80 @@
 {
-  "name": "pathology",
```

Comment:
Can you change this back to 2 spaces for indent? Make the lines changed in package-lock.json massive